_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -32,24 +33,101 @@ Our system is also able to send logs to the following SaaS log processing provid
 
 ## Configuration of your Application on Scalingo
 
-As this feature is private beta, you cannot activate logs drain by yourself. You need to add an
-environment variable containing the token to the used logs management service, and contact the
-support so that we activate the logs drain. In a matter of minutes, your logs flow to the logs
-management service.
+The log drains configuration will be done using the [CLI
+](https://doc.scalingo.com/cli).
+
+### Add a Log Drain
+
+To add a log drain management service you will have to use the `log-drains-add`
+command.
+Below, the list of available parameters:
+* `type`: Communication protocol
+* `url`: URL of self hosted ELK (optional)
+* `host`: Host of logs management service (optional)
+* `port`: Port of logs management service (optional)
+* `token`: Used by certain vendor for authentication (optional)
+* `drain-region`: Used by certain logs management service to identify the region
+of their servers (optional)
+
+The parameters used will depend on the chosen provider.
 
-The format of the environment variable depends on the provider:
+TODO(pc): check if we reference to elk self hosted documentation:
+https://doc.scalingo.com/platform/getting-started/getting-started-with-elk
 
-* Scalingo hosted ELK stack: `https://USER:TOKEN@logstash-app-name.osc-fr1.scalingo.io`
-* Datadog: `datadog://DD_API_KEY?region=REGION`. `DD_API_KEY` is the Datadog API key you can
-    find [here](https://app.datadoghq.com/account/settings#api). `REGION` value
+Below, an exhaustive list of examples of additions for each of the supported
+vendors
+
+* Scalingo hosted ELK stack:
```

Comment:
Displaying the different supported vendors as a list is not really sexy in the final output:https://documentation-service-pr968.osc-fr1.scalingo.io/platform/app/log-drain#add-a-log-drain

Reply:
I split the bullet list into multiple sections but I don't find the result really sexy yet.
Do you have any suggestions ?